setup.cfg

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python                                                           
+# -*- coding: utf-8 -*-                                                         
+#                                                                               
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.                          
+# SPDX-License-Identifier: BSD-2-Clause
+
+[metadata]
+name = tern
+author = VMWare Inc
+author-email = nishak@vmware.com
+summary = A compliance tool for containers
+description-file = README.md
+description-content-type = text/markdown; charset=UTF-8
+home-page = https://github.com/vmware/tern/
+project_urls =
+    Documentation = https://github.com/vmware/tern/tree/master/docs
+    Source Code = https://github.com/vmware/tern 
+    Issues = https://github.com/vmware/tern/issues
+license = BSD-2.0
+keywords = 
+    Distribution
+    Container
+    Cloud-Native
+classifier = 
+    Development Status :: 3 - Alpha
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Operating System :: POSIX
+    Operating System :: POSIX :: Linux
+    Operating System :: MacOS :: MacOS X
```

Comment:
We might want to get rid of this one since tern can't run natively on Mac OSX

Reply:
Removed.